common/ellipses.js

```diff
@@ -237,10 +243,12 @@
                                             // 2. Update UI by letting the table directive know
                                                 scope.$emit('record-modified');
                                             }).catch(function(error) {
+                                                // TODO: Throw the error instead
                                                 ErrorService.catchAll(error);
                                             });
                                         } else {
                                             scope.$emit('error', response);
+                                            // TODO: Throw the error instead
                                             ErrorService.catchAll(error);
                                         }
                                     }).catch(function (error) {
```

Comment:
[onSelect](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L266) should be wrapped